core/src/main/web/app/mainapp/components/notebook/codecell-directive.js

```diff
@@ -94,6 +94,15 @@
           return $scope.cellmodel.output.result !== undefined;
         };
 
+        $scope.getOutputHeight = function() {
```

Comment:
If you put a console.log here, you'll see that this function gets run a lot of times, we would want to avoid that if possible.